ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -6,25 +6,29 @@ import {Spinner} from 'app/components/spinners';
 import {runtimeApi} from 'app/services/swagger-fetch-clients';
 import colors, {addOpacity} from 'app/styles/colors';
 import {reactStyles, withCurrentWorkspace} from 'app/utils';
-import {allMachineTypes, validLeonardoMachineTypes} from 'app/utils/machines';
+import {allMachineTypes, validLeonardoMachineTypes, Machine} from 'app/utils/machines';
```

Comment:
The behavior of the panel will change before the final review.
1. A confirm panel will show up after a runtime change is requested
2. Once confirmed the page will navigate back to the preview mode

That behavior mirrors what is happening in Terra UI